### PR TITLE
Feat T30254 more adjustments for addons

### PR DESCRIPTION
**Pull request by bakridemos**
**Ticket:** https://yaits.demos-deutschland.de/T30254 

Description: more related addons adjustements

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php [hunk 1]
```diff
@@ -125,6 +122,7 @@ public function __construct(
      *     name="DemosPlan_plandocument_administration_element",
      *     path="/verfahren/{procedure}/verwalten/element/{elementId}",
      * )
+     *
      * @DplanPermissions("area_admin_paragraphed_document")
```
Comment (AlexMenschoid) [inline demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php:126]: @graupnerdemos Why is the CS-fixer now adding the empty line I wished for but couldn't get in the other PR?!

File: demosplan/DemosPlanCoreBundle/Entity/Document/Elements.php [hunk 2]
```diff
@@ -103,8 +105,11 @@ class Elements extends CoreEntity implements UuidEntityInterface
      * @var string|null
      *
      * @ORM\Column(name="_e_id", type="string", length=36, options={"fixed":true})
+     *
      * @ORM\Id
+     *
      * @ORM\GeneratedValue(strategy="CUSTOM")
+     *
```
Comment (AlexMenschoid) [inline demosplan/DemosPlanCoreBundle/Entity/Document/Elements.php:112]: @graupnerdemos And why is the CS Fixer in turn ripping these definitions apart?!

**Review by AlexMenschoid — COMMENTED** [review]
@graupnerdemos The CS Fixer is mean to me! I don't want to play CS Fixer ping-pong, that sounds like a very cruel form of developer hell....

@bakridemos otherwise the PR looks good, I just need to have the CS Fixer thing cleared, then I'll approve it.

File: demosplan/DemosPlanCoreBundle/Addon/FrontendAssetProvider.php [hunk 3]
```diff
@@ -66,6 +66,8 @@ public function getFrontendClassesForHook(string $hookName): array
 
             return $this->createAddonFrontendAssetsEntry($hookData, $assetContents);
         }, $this->registry->getAddonInfos());
+
+        return array_filter($assetList, fn (array $assetInfo) => 0 !== count($assetInfo));
```
Comment (dresslerdemos) [inline demosplan/DemosPlanCoreBundle/Addon/FrontendAssetProvider.php:70]:
```suggestion

        // avoid exposing addon information unnecessarily
        return array_filter($assetList, fn (array $assetInfo) => 0 !== count($assetInfo));
```

**Review by dresslerdemos — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Event/Procedure/PostNewProcedureCreatedEvent.php [hunk 4]
```diff
@@ -43,4 +45,10 @@ public function getToken(): ?string
     {
         return $this->token;
     }
+
+    public function addCriticalConcern(string $key, string $eventConcernText, Exception $e)
+    {
+        $eventConcern = new EventConcern($eventConcernText, $e);
+        $this->addCriticalEventConcern($key, $eventConcern);
+    }
```
[the remark this answers is not in the document]
Comment (bakridemos) [reply demosplan/DemosPlanCoreBundle/Event/Procedure/PostNewProcedureCreatedEvent.php]: done in https://github.com/demos-europe/demosplan-core/pull/721/commits/afd2690b8e57e1fec2e2affc52deb89c4d780822

**Review by dresslerdemos — APPROVED** [review]

**Review by dresslerdemos — APPROVED** [review]

**Pull request merged**